### PR TITLE
docs(ovh-shell): add ovh shell docs

### PR DESCRIPTION
**Pull request by zakariasahmane**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| License          | BSD 3-Clause

## Description

Add documentation for `ovh-shell` library and the container.

### CODE REVIEW

**Review by Jisay — APPROVED** [review]

File: packages/components/ovh-shell/README.md [hunk 1]
```diff
@@ -1 +1,200 @@
...
+This plugin handles everything that is UI related, such as elements shown or hidden, the state or position of certain elements etc.
+Currently, it handles the state of the account sidebar, notifications sidebar and verifies cookie verification for the ssoAuthModal.
+
+- `showAccountSidebar() => void`) : display the account sidebar.
+- `disableAccountSidebarToggle() => void`) : Disallows toggle of the Account Sidebar. After calling this function, the Account Sidebar won't toggle anymore unless you call `enableAccountSidebarToggle`.
+- `enableAccountSidebarToggle() => void`) : Allows Account sidebar to be toggled.
+- `isAccountSidebarVisible() => boolean`) : Returns if Account Sidebar is shown or not.
+- `setForceAccountSiderBarDisplayOnLargeScreen(isForced: boolean) => void`) : forces the display on large screens. This option can be disabled by passing the false parameter to `isFalse`.
+- `resetAccountSidebar() => void` : Sets the Account Sidebar to its default state.
+- `isMenuSidebarVisible() => boolean` : Returns if the PNR menu is displayed.
```
Comment (antleblanc) [inline packages/components/ovh-shell/README.md:156]: **question:** Shouldn't we found a name for this menu? PNR sounds more like a project name.

**Review by antleblanc — CHANGES_REQUESTED** [review]
That's a really good start!
Just left some comments regarding naming/punctuation (mainly).

File: docs/docs/guide/applications.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ The monolithic applications can be started for a specific region
 | [Bare Metal Cloud](#bare-metal-cloud) | EU / CA / US |
 | [Public Cloud](#public-cloud)         | EU / CA / US |
 | [Telecom](#telecom)                   | EU           |
+| [Container](#container)               | EU / CA / US |
```
[the remark this answers is not in the document]
Comment (zakariasahmane) [reply docs/docs/guide/applications.md:21]: That's actually a very good question, i don't know if it should be added to the list of monolithic applications or have a section of its own. 

Maybe @frenautvh @Jisay or @cbourgois can help answer this question?

File: docs/docs/guide/architecture.md [hunk 3]
```diff
@@ -21,22 +22,65 @@ Several [workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) are configure
...
 ## Build
 
-[webpack](https://webpack.js.org/) and [rollup.js](https://rollupjs.org) are respectively used to build our applications, modules and components.
+[webpack](https://webpack.js.org/), [rollup.js](https://rollupjs.org) and [Vite](https://vitejs.dev/) are respectively used to build our applications, modules and components.
 
 You can see more about our tooling [here](/guide/tools.md).
+
+## Micro-FrontEnd
+
+In order to enable the uFrontEnd architecture, we use what we call a `Container`. The container contains a standalone application and elements that are common to all applications.
```
[the remark this answers is not in the document]
Comment (zakariasahmane) [reply docs/docs/guide/architecture.md:64]: @antleblanc I actually agree with you, i just didn't know how to type that character in my Mac and i thought, i'll copy it !

**Pull request merged**